examples/gno.land/r/nft/nft.gno

```diff
@@ -5,80 +5,43 @@ import (
 	"strconv"
 
 	"gno.land/p/avl"
+	igrc721 "gno.land/p/grc/grc721"
```

Comment:
Is it customary to name it "i"+name ? 
I've been appending "m" instead, for "module". but I suppose not all packages are modules so...

Reply:
the `i` prefix is common for packages that only define interfaces

to be honest, I opened this PR early with this first split, but I'm considering more and more to not only push the interface but the interface + a minimal (official?) implementation, so anyone can import it and use the implementation, or import it to ensure having compatibility with the interface

no strong conviction on the naming yet, it's up to you :)

Reply:
the `i` prefix is common for packages that only define interfaces

to be honest, I opened this PR early with this first split, but I'm considering more and more to not only push the interface but the interface + a minimal (official?) implementation, so anyone can import it and use the implementation, or import it to ensure having compatibility with the interface

no strong conviction on the naming yet, it's up to you :)

Reply:
if it's stateless, it can still live in /p/grp, and it can be known to be "pure". so as long as the package doesn't have global variables, all good.